### DISCUSSION TITLE
Tasmocompiler 9.0.1 fails on ESP32 Generic compilation

### DISCUSSION THREAD

**Discussion by helmutoh** [benzino77/tasmocompiler#292] [Q&A] [answered]
Hello! 

I have been trying to compile Tasmota for ESP32-Generic using latest tasmo compile 9.0.1

It always fails with error code #1: *** [.pio/build/tasmota32/src/tasmota.ino.cpp.o] Error 1

I have used new workspace and tried all versions of tasmota but no success.

Trying to compile 8266 image works fine ... 

any ideas?

**Comment by helmutoh** [comment]
Found some hint, tried always to compile with the old "scripts" support. Seems that was causing some conflict. Using Berry Scripts instead makes it run through ...

**Comment by benzino77** [comment, marked as answer]
Hi there!

It works fine for me:
![2022-07-24_14-49](https://user-images.githubusercontent.com/15658235/180647755-ec6b85cd-4a17-4d56-8974-7e8af01130d6.png)

1. If you are trying to compile development branch of Tasmota, you should try to use development branch of TasmoCompiler
2. Default features selected for each board are "rather mandatory", and you should "uncheck" them with caution
3. Some features are incompatible with each other - in such case you have to ask Tasmota Dev Team what is the reason.
